### PR TITLE
Add code of conduct page

### PR DESCRIPTION
**Pull request by judithpatudith**
Adding code of conduct page and menu items on both purple and white footers

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).

### CODE REVIEW

File: src/coc.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+
+The Mesosphere DC/OS community is made up of users and contributors from around the world who collaborate in chat, on mailing lists, at events, and on other channels to help each other get the most out of DC/OS.
+
+Everyone you encounter in chat or on the mailing list is contributing voluntarily and helps on a best-effort basis, including Mesosphere employees. Treating everyone in the project with respect is essential to make sure members of our community feel welcome and encouraged to participate in the future. Please be kind and patient with your fellow community members.
+
+Additionally DC/OS has a code of conduct that participants are expected to adhere to in order to make sure that no one is discriminated against or excluded because of their identity.
+
+Breaches of the code of conduct and inappropriate behavior are handled on a case by case basis, but the consequences for violating it may include limiting or removing your ability to collaborate on project resources and at events.
+
+## Contributor Covenant Code of Conduct
```
Comment (pleia2) [inline src/coc.md]: We can drop the words "Contributor Covenant" (since it has been modified, and modification is the point) and update it to say "Mesosphere DC/OS Community Code of Conduct"

Comment (pleia2) [reply src/coc.md]: Could drop "Community" - it's getting a bit wordy :)

Comment (judithpatudith) [reply src/coc.md]: How about "DC/OS Code of Conduct"? Even shorter, and we only have to include "Mesosphere" in the first mention.

Comment (pleia2) [reply src/coc.md]: Ah yes, perfect.

**Comment by jmanalus** [comment]
Hey @judithpatudith - I've tested the build locally and everything looks great. If you're happy with the content then I can merge it into master.

**Comment by pleia2** [comment]
We'll need a quick follow-up patch to this one to fix up the CoC title I noted before the site is promoted.
